x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -595,6 +598,37 @@ export const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
           },
         ],
       },
+      '3': {
+        changes: [
+          {
+            type: 'mappings_addition',
+            addedMappings: {
+              es_index_patterns: {
+                dynamic: false,
+                properties: {
+                  pattern: { type: 'keyword' },
```

Comment:
**doubt**: isn't it mismatching with the backfilled `title` property later used in the `backfillFn`function?
Also, should we keep it aligned and update the mapping with `title` instead of `pattern`?

Reply:
I see some other places in the PR where `pattern` is still used in place of `title`, like in some tests and the current_mappings file.

Reply:
urgh, went through these.